newsfragments/527.internal.md

```diff
@@ -0,0 +1 @@
+Make sure MAS Secret & ConfigMap hooks are created only when required.
```

Comment:
Given `syn2mas` hasn't been in a release yet, let's just c/p that changelog fragment here as this is a fix-up to that PR. Then both PRs will be linked to the changelog entry